### DISCUSSION TITLE
4차 남은 스프린트 TODO

### DISCUSSION THREAD

**Discussion by jminkkk** [woowacourse-teams/2024-code-zap#481] [이야기해봐요]
## 남은 스프린트 TODO

- 타 회원의 템플릿 보기 기능 추가 

### 구경가기

- 비회원도 조회가능

---

팀 로고("/") -> 서비스 소개(랜딩 페이지, 홈페이지, about us)

**Comment by kyum-q** [comment]
## 타 회원의 템플릿 보기 기능 구체화
- 구경가기 페이지
- 전체 회원 템플릿 조회
- 정렬 (최신순/오래된순) 

## 비회원도 구경가기 가능한가?

**Comment by zeus6768** [comment]
# 타 회원의 템플릿 보기 → "구경가기"
# 둘러보기 → "구경가기"
